dockers/docker-database/database_config.json.j2

```diff
@@ -2,9 +2,10 @@
     "INSTANCES": {
         "redis":{
             "hostname" : "{{HOST_IP}}",
-            "port" : 6379,
-            "unix_socket_path" : "/var/run/redis{{NAMESPACE_ID}}/redis.sock",
-            "persistence_for_warm_boot" : "yes"
+            "port" : {{REDIS_PORT}},
+            "unix_socket_path" : "/var/run/redis{{DEV}}/redis.sock",
+            "persistence_for_warm_boot" : "yes",
+            "database_type": "{{DATABASE_TYPE}}"
```

Comment:
should we add a check to add the field "database_type" if DATABASE_TYPE is defined?

Reply:
Done